src/cli.ts

```diff
@@ -15,13 +15,15 @@ async function buildCommands() {
     .option('external', { type: 'array', nargs: 1, default: [], desc: 'Packages in this list will be excluded from the bundle and added as dependencies (example: fsevents:optional)' })
     .option('allowed-license', { type: 'array', nargs: 1, default: [], desc: 'List of valid licenses' })
     .option('resource', { type: 'array', nargs: 1, default: [], desc: 'List of resources that need to be explicitly copied to the bundle (example: node_modules/proxy-agent/contextify.js:bin/contextify.js)' })
-    .option('dont-attribute', { type: 'string', desc: 'Dependencies matching this regular expressions wont be added to the notice file' })
+    .option('dont-attribute', { type: 'string', desc: 'Dependencies matching this regular expressions wont be added to the attribution files' })
     .option('test', { type: 'string', desc: 'Validation command to sanity test the bundle after its created' })
     .command('validate', 'Validate the package is ready for bundling', args => args
       .option('fix', { type: 'boolean', default: false, alias: 'f', desc: 'Fix any fixable violations' }),
     )
     .command('write', 'Write the bundled version of the project to a temp directory')
-    .command('pack', 'Write the bundle and create the tarball')
+    .command('pack', 'Write the bundle and create the tarball', args => args
+      .option('target', { type: 'string', desc: 'Target directory of the tarball' }),
```

Comment:
I noticed this option was missing from the CLI, so decided to add it here since it going to be needed fairly soon.